apps/emqx_ldap/src/emqx_ldap.erl

```diff
@@ -114,6 +114,28 @@ ensure_username(required) ->
 ensure_username(Field) ->
     ?ECS:username(Field).
 
+parse_config(Config, ToKeep, ToString) ->
+    Convert = fun(Value) ->
+        case lists:member(Value, ToString) of
```

Comment:
It seems we should check if this `Value`'s `Key` is in the `ToString` list, rather than the value itself?